webapp/.env.template

```diff
@@ -1,5 +1,6 @@
 SENTRY_DSN_WEBAPP=
 COMMIT=
 PUBLIC_REGISTRATION=false
+INVITE_REGISTRATION=true
```

Comment:
Do we really want this set to `true` in the template? I think it is better to set it to `false`